setup.cfg

```diff
@@ -1,3 +1,3 @@
 [flake8]
-ignore = E501,E226,E731,F405
+ignore = E501,E226,E731,F405,F403,F401
```

Comment:
Yes, `__init__.py` is an exception, you should add the comment `  # noqa` at the end of the import line to disable flake8 errors/warnings just on that line.